rdrf/registry/patients/models.py

```diff
@@ -1008,14 +1008,67 @@ def family(self):
 
 
 class ClinicianOther(models.Model):
+    use_other = models.BooleanField(default=False)
     patient = models.ForeignKey(Patient, null=True)
-    clinician_name = models.CharField(max_length=200, null=True)
-    clinician_hospital = models.CharField(max_length=200, null=True)
-    clinician_address = models.CharField(max_length=200, null=True)
+    clinician_name = models.CharField(max_length=200, blank=True,null=True)
+    clinician_hospital = models.CharField(max_length=200, blank=True, null=True)
+    clinician_address = models.CharField(max_length=200, blank=True, null=True)
     clinician_email = models.EmailField(max_length=254, null=True, blank=True)
     clinician_phone_number = models.CharField(max_length=254, null=True, blank=True)
+    user = models.ForeignKey(CustomUser, blank=True, null=True)
+
+    def synchronise_working_group(self):
+        if self.user:
```

Comment:
I'd probably flip this `if` and return from the function if self.user is None, then you can drop the indent and anyone adding code won't accidentally create a security issue

Reply:
will do